src/services/github.ts

```diff
@@ -1,3 +1,6 @@
+import dotenv from "dotenv";
+dotenv.config();
```

Comment:
Not sure if that a good idea to load dotenv here (should not be the responsability of that package).